packages/worker/.env.dist

```
@@ -0,0 +1 @@
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/postgres"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)